apps/src/StudioApp.js

```diff
@@ -1456,7 +1456,7 @@ StudioApp.prototype.configureDom = function (config) {
     visualizationColumn.className = visualizationColumn.className + " embed_hidesource";
   }
 
-  if (!config.embed && !config.hideSource) {
+  if (!config.share) {
```

Comment:
Do you really want the responsive styles to stay enabled in your new "hideSource only" mode? (they are mostly used to squeeze the visualization horizontally with breakpoints assuming that there is source on the right hand side)